database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,11 +15,10 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('username');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
-            $table->rememberToken();
```

Comment:
why are `rememberToken` and `email_verified_at` field removed.
they do have specific purpose.
We Will need and may use later on in the course